content/en/blog/2023/ambient-ebpf-redirection/index.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Leverage eBPF for traffic redirection in Istio ambient mode"
+description: An alternative approach to redirect application pod traffic to ztunnel in Istio ambient mode.
+publishdate: 2023-03-15
+attribution: "Iris Ding (Intel), Chun Li (Intel)"
+keywords: [istio,ambient,ztunnel,eBPF]
+---
+
+The istio-cni component running on each Kubernetes worker node is responsible for redirecting application pod traffics to ztunnel on that node. By default it relies on iptables and
+Geneve tunnel to achieve this redirection.  Now a new approach which is based on eBPF is also available in Istio ambient mode for this purpose.
+
+## Why eBPF

```

Comment:
Also need "Why not eBPF? If the new eBPF way is better, why isn't it the default implementation?

Using eBPF (or not) to redirect the traffic seems like an implementation detail. Why are you presenting this feature as a user choice? If it's just because it's a potentially buggy prototype, that will become the default in the future, this blog post should say that and also list any current limitations. If on the other hand there will be 2 implementations for the foreseeable future, then we need to clearly list the pros and cons of both implementations.

Reply:
will add a section about pros and cons